ToyBox/classes/Models/Settings.cs

```diff
@@ -118,8 +118,6 @@ public class Settings : UnityModManager.ModSettings {
         public bool toggleRotateOnAllMaps = false;
         public bool toggleScrollOnAllMaps = false;
         public bool toggleCameraPitch = false;
-        public bool toggleCameraElevation = false;
```

Comment:
these changes should not be deleted.  Please rebase and and apply your patch to top of tree.